protoc-maven-plugin/src/main/java/io/github/blackrock/protocjar/maven/ProtocJarMojo.java

```diff
@@ -258,6 +258,15 @@ public class ProtocJarMojo extends AbstractMojo
 	@Parameter(name="protocArtifact")
 	private String protocArtifact;
 
+	/**
+	* This parameter (when set to false) allows to execute the protoc code generation
```

Comment:
I also suggest we simplify the JavaDoc and make it easier to follow. Such as “Set parameter to false from default true to override the default ‘pom’ packaging skipping logic, when working with non-Java languages. The default logic is required for standard Java packaging like JAR, WAR and EAR“.